django_simple_bulma/templatetags/django_simple_bulma.py

```diff
@@ -30,7 +30,7 @@ def bulma(theme: str = "") -> SafeString:
     css = static(f"css/{theme + '_' if theme else ''}bulma.css")
     html = [
         f'<link rel="preload" href="{css}" as="style">',
-        f'<link rel="stylesheet" href="{css}">',
+        f'<link rel="stylesheet" href="{css}" id="bulma-css{"" if theme == "" else f"-{theme}"}">',
```

Comment:
maybe you can refactor this so it doesn't give people a heart attack? 🌞 

Maybe this id can be created on an earlier line so that we don't have to have an f string within an f string with a ternary conditional, that's a lot of logic to stuff into one line.